Gemfile-rails-dependencies

```diff
@@ -17,9 +17,9 @@ when /stable$/
 when nil, false, ""
   if RUBY_VERSION < '1.9.3'
     # Rails 4+ requires 1.9.3+, so on earlier versions default to the last 3.x release.
-    gem "rails", "~> 3.2.17"
+    gem "rails", "~> 3.2"
   else
-    gem "rails", "~> 4.0.4"
+    gem "rails", "~> 4.0"
```

Comment:
This is for the default Rails version someone will use when setting up the repo. I agree we probably don't want this to be `~> 4.0`, but we probably should bump this to either `~> 4.1.0` or `~> 4.2.0`. My vote is for the latter.